Exercises/arrays-methods/arrays-methods.js

```diff
@@ -0,0 +1,24 @@
+// Task #1
+const array = [1, 2, 3, 4];
+const isElementPresent = (ar, element) => ar.includes(element);
+console.log(isElementPresent(array, 3));
+
+// Task #2
+const array = [2, 4];
+const isArrayEven = (ar) => Boolean(!ar.filter(item => item % 2 === 1).length);
```

Comment:
You don't need `Boolean()` here because `!` converts length to boolean.
Also it's ok to use `filter` but it would be more simple to use `every` or `some` method.